scrapers/AussieAss.yml

```diff
@@ -0,0 +1,24 @@
+name: AussieAss
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.aussieass.com
+      - https://www.aussiepov.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class='col-lg-12 col-md-12 col-sm-12 col-xs-12 preview-description']/h4/span/text()
+      Performers:
+        Name: 
+          selector: //div[@class='col-lg-12 col-md-12 col-sm-12 col-xs-12 preview-description']/h5/span/a/text()
+      Image:
+        selector: //img[@class='large_update_thumb left thumbs']/@src
+        postProcess:
+          - replace:
+            - regex: ^
+              with: https://www.aussieass.com/
```

Comment:
This replace only works for 1 of the two sites, on aussiepov, this creates a broken link. 

This is ridiculously hacky but I got it to work with:

```YAML
      Image:
        selector: //div[@class='navbar-header']/a/@href|//img[@class='large_update_thumb left thumbs']/@src
        concat: " "
        postProcess:
          - replace:
            - regex: (.+\.com)(?:\s|\/.+\s)(.+)
              with: $1/$2
```

Basically it makes it so we match a field with the site URL and the image URL, concat them separated by a space, then clean/reassemble them together (each site had the URL slightly different) to make a site specific image URL.